pkg/cli/config/config_test.go

```diff
@@ -79,27 +79,27 @@ func TestBackplaneConfiguration_getFirstWorkingProxyURL(t *testing.T) {
 		},
 		{
 			name:    "valid-proxies",
-			proxies: []string{"https://dummy.com"},
+			proxies: []string{"https://proxy.invalid"},
 			clientDoFunc: func(client *http.Client, req *http.Request) (*http.Response, error) {
 				return &http.Response{StatusCode: http.StatusOK}, nil
 			},
-			want: "https://dummy.com",
+			want: "https://proxy.invalid",
 		},
 		{
 			name:    "multiple-valid-proxies",
-			proxies: []string{"https://dummy.com", "https://dummy.proxy"},
+			proxies: []string{"https://proxy.invalid", "https://dummy.proxy.invalid"},
```

Comment:
Looks the "https://dummy.proxy.invalid" still contains `dummy" keyword, should it be replaced as well?

Reply:
Looking at the Jira card, I think we're okay here. The concern was that `dummy[.]com` is a malicious website that we don't want to be pinging, so as long we don't directly hit that URL we should be fine.